src/stdio_mgr/stdio_mgr.py

```diff
@@ -191,6 +205,13 @@ def stdio_mgr(in_str=""):
     sys.stdout = old_stdout
     sys.stderr = old_stderr
 
-    new_stdin.close()
+    try:
+        closed = new_stdin.closed
+    except ValueError:
+        pass
+    else:
+        if not closed:
```

Comment:
Similarly here -- if I read that same [docs bit](https://docs.python.org/3/library/io.html#io.IOBase.close) correctly, this guard clause on the `.close()` call is unnecessary: calling `.close()` on an already-closed `TextIOWrapper` should just be a no-op?

Reply:
Not when detached.  py docs maybe need enhancement.